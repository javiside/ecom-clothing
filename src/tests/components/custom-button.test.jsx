import React from "react";
import { shallow } from "enzyme";

import CustomButton from "../../components/custom-button";

describe("CustomButton", () => {
  it("should mount the component correctly", () => {
    expect(shallow(<CustomButton />)).toMatchSnapshot();
  });
});
