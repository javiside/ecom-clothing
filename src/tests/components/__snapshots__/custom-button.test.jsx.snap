// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomButton should mount the component correctly 1`] = `
<button
  className="custom-button"
/>
`;
